src/frontend/next/out/404.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html lang="en"><head><link rel="icon" href="/logo.svg" type="image/svg+xml"/><meta name="viewport" content="width=device-width"/><meta charSet="utf-8"/><title>404: This page could not be found</title><meta name="next-head-count" content="3"/><link rel="preload" href="/_next/static/css/e0b9833c064efc2eda42.css" as="style"/><link rel="stylesheet" href="/_next/static/css/e0b9833c064efc2eda42.css" data-n-g=""/><noscript data-n-css=""></noscript><link rel="preload" href="/_next/static/chunks/main-0db40ba1b1ffc82919b2.js" as="script"/><link rel="preload" href="/_next/static/chunks/webpack-50bee04d1dc61f8adf5b.js" as="script"/><link rel="preload" href="/_next/static/chunks/framework.a4b3ed6c46c358b601ab.js" as="script"/><link rel="preload" href="/_next/static/chunks/f6078781a05fe1bcb0902d23dbbb2662c8d200b3.d4f57090bd55662b89aa.js" as="script"/><link rel="preload" href="/_next/static/chunks/pages/_app-bde721f06a5022f001d6.js" as="script"/><link rel="preload" href="/_next/static/chunks/pages/_error-e2ffa4674eea62b50f3f.js" as="script"/><style id="jss-server-side"></style></head><body><div id="__next"><div style="color:#000;background:#fff;font-family:-apple-system, BlinkMacSystemFont, Roboto, &quot;Segoe UI&quot;, &quot;Fira Sans&quot;, Avenir, &quot;Helvetica Neue&quot;, &quot;Lucida Grande&quot;, sans-serif;height:100vh;text-align:center;display:flex;flex-direction:column;align-items:center;justify-content:center"><div><style>body { margin: 0 }</style><h1 style="display:inline-block;border-right:1px solid rgba(0, 0, 0,.3);margin:0;margin-right:20px;padding:10px 23px 10px 0;font-size:24px;font-weight:500;vertical-align:top">404</h1><div style="display:inline-block;text-align:left;line-height:49px;height:49px;vertical-align:middle"><h2 style="font-size:14px;font-weight:normal;line-height:inherit;margin:0;padding:0">This page could not be found<!-- -->.</h2></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"statusCode":404}},"page":"/_error","query":{},"buildId":"A9UYMyJSJ3cx9OK58xeoM","nextExport":true,"isFallback":false,"gip":true}</script><script nomodule="" src="/_next/static/chunks/polyfills-4acaa4b69e5977b3ab87.js"></script><script src="/_next/static/chunks/main-0db40ba1b1ffc82919b2.js" async=""></script><script src="/_next/static/chunks/webpack-50bee04d1dc61f8adf5b.js" async=""></script><script src="/_next/static/chunks/framework.a4b3ed6c46c358b601ab.js" async=""></script><script src="/_next/static/chunks/f6078781a05fe1bcb0902d23dbbb2662c8d200b3.d4f57090bd55662b89aa.js" async=""></script><script src="/_next/static/chunks/pages/_app-bde721f06a5022f001d6.js" async=""></script><script src="/_next/static/chunks/pages/_error-e2ffa4674eea62b50f3f.js" async=""></script><script src="/_next/static/A9UYMyJSJ3cx9OK58xeoM/_buildManifest.js" async=""></script><script src="/_next/static/A9UYMyJSJ3cx9OK58xeoM/_ssgManifest.js" async=""></script></body></html>
```

Comment:
Can you remove the `out` folder too? `git add .` is generally not your friend 😋

Reply:
I removed out/cache folders, commited it but it just created another branch, not to this PR 😀